__tests__/functional/client-configuration.test.ts

```diff
@@ -60,7 +60,7 @@ an environmental variable named FAUNA_SECRET or pass it to the Client constructo
     expect(client.client.defaults.baseURL).toEqual("http://localhost:7443/");
     const result = await client.query<number>({ query: '"taco".length' });
     expect(result.txn_time).not.toBeUndefined();
-    expect(result).toEqual({ data: 4, txn_time: result.txn_time });
+    expect(result).toEqual({ data: 4, txn_time: result.txn_time, summary: "" });
```

Comment:
This seems like odd behavior from Core.

@freels @macmv is it intended that core return empty strings for summary when there's no summary data?

Reply:
No, it probably shouldn’t do that. In general my philosophy too is that the drivers shouldn’t be testing core behavior like this, so if possible I’d tweak it to only check the data field itself or something.